slsSupportLib/include/sls/bit_utils.h

```diff
@@ -18,4 +19,76 @@ template <typename T> std::vector<int> getSetBits(T val) {
     }
     return set_bits;
 }
+
+class RegisterAddress {
+  private:
+    uint32_t addr_{0};
+
+  public:
+    constexpr RegisterAddress() noexcept = default;
+    explicit RegisterAddress(uint32_t address) : addr_(address) {}
+    explicit RegisterAddress(const std::string &address);
+
+    std::string str() const;
+    operator uint32_t() const noexcept { return addr_; }
+
+    constexpr bool operator==(const RegisterAddress &other) const {
+        return (addr_ == other.addr_);
+    }
+    constexpr bool operator!=(const RegisterAddress &other) const {
+        return (addr_ != other.addr_);
+    }
+} __attribute__((packed));
```

Comment:
Do you have a specific reason to use attribute packed? With a single member it doesn't make a different except that it changes the alignment from 4 to 1 bytes.